docs/design/operator-bundle.md

```diff
@@ -116,157 +121,13 @@ In order to use `opm` CLI, follow the `opm` build instruction:
 1. Clone the operator registry repository:
 
 ```bash
-$ git clone https://github.com/operator-framework/operator-registry
+git clone https://github.com/operator-framework/operator-registry
 ```
 
 2. Build `opm` binary using this command:
 
 ```bash
-$ make build
+make build
 ```
 
 Now, a binary named `opm` is now built in current directory and ready to be used.
-
```

Comment:
Is there a reason to remove this section?

Reply:
I was just going based on what Kevin was saying in https://github.com/operator-framework/operator-registry/pull/546#discussion_r562758419

Reply:
I see. I commented on the PR that Kevin mentioned. I'm against removing those alpha commands from `opm`. They are under `alpha` and not visible for a reason as they are meant for internal/testing/development use. We don't promote those commands as well because SDK is the official tool.
Those commands are frankly a way for us to rapid-test the changes we make to the library we have in registry repo.
Either way, it is OK to remove that section for now. We can always add it back.